src/libraries/System.Private.CoreLib/src/System/Reflection/Emit/SymbolType.cs

```diff
@@ -271,18 +271,30 @@ public override Type MakeByRefType()
             return FormCompoundType(_format + "&", _baseType, 0)!;
         }
 
+        [RequiresDynamicCode("The code for an array of the specified type might not be available.")]
```

Comment:
Were these RDC attributes added due to warnings?

Reply:
Yes, a warning after referencing the type in ref emit